### PR TITLE
bsp: xilinx-versal: add support for vck190-versal

### PR DESCRIPTION
**Pull request by ricardosalveti**
Signed-off-by: Ricardo Salveti <ricardo@foundries.io>

### CODE REVIEW

**Review by MrCry0 — APPROVED** [review]
LGTM

File: bsp/xilinx-versal/versal.cfg [hunk 1]
```diff
@@ -0,0 +1,313 @@
...
+CONFIG_SERIAL_AMBA_PL010=y
+CONFIG_SERIAL_AMBA_PL010_CONSOLE=y
+CONFIG_SERIAL_AMBA_PL011=y
+CONFIG_SERIAL_AMBA_PL011_CONSOLE=y
+
+#
+CONFIG_I2C=y
+CONFIG_I2C_MUX=y
+CONFIG_I2C_MUX_PCA954x=y
+CONFIG_I2C_MUX_REG
```
Comment (vanmaegima) [inline bsp/xilinx-versal/versal.cfg:66]: shouldn't this be?
```suggestion
CONFIG_I2C_MUX_REG=y
```

Comment (ricardosalveti) [reply bsp/xilinx-versal/versal.cfg:66]: Yes, nice catch.

**Review by vanmaegima — APPROVED** [review]
LGTM

**Pull request merged**